src/components/shared/LessonSide/LevelSelector.tsx

```diff
@@ -1,5 +1,42 @@
-function LevelSelector(): JSX.Element {
-  return <div>Level Selector</div>;
+import '../../../styles/LessonSide.scss';
+
+interface LevelSelectorProps {
+  numLevels: number;
+  currentLevel: number;
+}
```

Comment:
Can we add a prop about `maxLevelReached` so that the connecting lines are visible up to the `maxLevelReached`